pyflakes/checker.py

```diff
@@ -1687,19 +1691,68 @@ def CALL(self, node):
                 self.handleNode(node.args[0], node)
 
         elif _is_typing(node.func, 'TypeVar', self.scopeStack):
+
             # TypeVar("T", "int", "str")
-            for arg in node.args[1:]:
-                if isinstance(arg, ast.Str):
-                    with self._enter_annotation():
-                        self.handleNode(arg, node)
+            omit += ["args"]
+            annotated += [arg for arg in node.args[1:] if isinstance(arg, ast.Str)]
```

Comment:
even if they're not strings we need to treat them as annotations -- for instance you might have `List["T"]`